packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -375,23 +375,29 @@ class RenderEditable extends RenderBox {
     return _textPainter.maxIntrinsicWidth;
   }
 
-  // This does not required the layout to be updated.
-  double get _preferredLineHeight => _textPainter.preferredLineHeight;
+  /// The height of a typical text line in logical pixels.
+  ///
+  /// This height is based on properties of the text's style such as
+  /// font size etc and could change when those properties change.
+  /// This does not require the layout to be updated.
```

Comment:
Why does it not require the layout to be updated? e.g. if the font size was animated up, wouldn't the toolbar need to animate its position also?